pkg/build/builder/sti.go

```diff
@@ -40,7 +41,11 @@ type runtimeBuilderFactory struct{}
 
 // Builder delegates execution to S2I-specific code
 func (_ runtimeBuilderFactory) Builder(config *s2iapi.Config, overrides s2ibuild.Overrides) (s2ibuild.Builder, s2iapi.BuildInfo, error) {
-	builder, buildInfo, err := s2i.Strategy(config, overrides)
+	client, err := docker.NewEngineAPIClient(config.DockerConfig)
```

Comment:
A comment somehow connecting the dots to the docker daemon load issue might be useful months from now when someone looks at this and asks why we don't construct the client just once.